R/pkg/tests/fulltests/test_mllib_tree.R

```diff
@@ -225,7 +225,7 @@ test_that("spark.randomForest", {
   expect_error(collect(predictions))
   model <- spark.randomForest(traindf, clicked ~ ., type = "classification",
                              maxDepth = 10, maxBins = 10, numTrees = 10,
-                             handleInvalid = "skip")
+                             handleInvalid = "keep")
```

Comment:
Because of R always ```forceIndexLabel``` which will index label whether it is numeric or string type, this leads to ```0.0``` and ```0``` in R label are different. If we choose ```skip```, it will make all labels unseen. I think this is a bug, maybe we should fix it in a separate PR.